traffic_portal/app/src/common/modules/table/typeDeliveryServices/table.typeDeliveryServices.tpl.html

```diff
@@ -26,7 +26,6 @@
 		columns="columns"
 		drop-down-options="dropDownOptions"
 		context-menu-options="contextMenuOptions"
-		sensitive-columns="sensitiveColumns"
```

Comment:
I didn't check this one but with two whole points of data I'm confident that this is broken too